openshift/patches/0003-quarkus-productized.patch

```diff
@@ -7,7 +7,7 @@ index 2bc2df55..e18ac0f3 100644
      <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
      <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
 -    <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
--    <quarkus.platform.version>2.15.0.Final</quarkus.platform.version>
+-    <quarkus.platform.version>2.15.2.Final</quarkus.platform.version>
 +    <quarkus.platform.group-id>com.redhat.quarkus.platform</quarkus.platform.group-id>
 +    <quarkus.platform.version>2.7.6.Final-redhat-00006</quarkus.platform.version>
```

Comment:
@matejvasek the released/supported downstream version is now 2.13 (https://access.redhat.com/documentation/en-us/red_hat_build_of_quarkus/rhbq-documentation-2-13/guide/0f24d6b4-7032-4601-99cb-fbdefec89f6d), can we use that instead?